### PR TITLE
create recipe for libuldaq

### PR DESCRIPTION
**Pull request by jdowner**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml"
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by jdowner** [comment]
I'm willing to have libuldaq listed

File: recipes/libuldaq/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+
+build:
+  skip: True  # [win]
+  number: 0
+  script: |
+    libtoolize
+    autoreconf -i
+    ./configure --disable-examples --prefix=$PREFIX
+    make -j"${CPU_COUNT}"
+    make install
```
Comment (scopatz) [inline recipes/libuldaq/meta.yaml:23]: Please put this in a `build.sh`

Comment (jdowner) [reply recipes/libuldaq/meta.yaml]: Fixed.

File: recipes/libuldaq/build.sh [hunk 2]
```diff
@@ -0,0 +1,7 @@
+#!/bin/sh
+
+libtoolize
+autoreconf -i
+./configure --disable-examples --prefix=$PREFIX
+make -j"${CPU_COUNT}"
+make install
```
Comment (scopatz) [inline recipes/libuldaq/build.sh:7]: Is there a `make check` that can be run?

Comment (jdowner) [reply recipes/libuldaq/build.sh:7]: Sure! Fixed.

**Comment by scopatz** [comment]
Thanks!

**Pull request merged**